### PR TITLE
Final Documentation PR

### PR DESCRIPTION
**Pull request by grossamos**
This is the changelog for the final documenation.

Please check:
- [ ] That your story point count is correct
- [ ] You are happy with your ``sonstige Beiträge``
- [ ] You couldn't find any more Spelling mistakes

Note @jeppard , Requirements was chosen instead of ``Anforderungen``, because this was the term used by Garidis
Not 2.0, this will be merged as soon as @Sophie152 finishes her final task (and the PR by @jeppard is reviewed)

### CODE REVIEW

**Comment by LukasLJL** [comment]
Maybe delete ``Formelverzeichnis`` & ``Listings``.
In Figura 2.1 ``Jonas Eppart`` -> ``Jonas Eppard``
There is an empty Page at B (119), G (124)
Max. Page Counter is wrong I guess.

File: content/06kapitel.tex [hunk 1]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+Die Beiträge der einzelnen Personen sind in Tabelle \ref{tab:person_beitrag} nachzulesen.
+Hierbei ist hervorzuheben, dass bei vielen Teammitgliedern Reviews und Fachliche Ausarbeitungen den Großteil deren Mitarbeit dargestellt hat, aber nicht mit in die Story Point Summe miteingeflossen ist. 
+Diese Sonstigen Beiträge sind jedoch in der entsprechenden Spalte in der Tabelle nachzulesen
+
+\begin{table}[h]
+    \resizebox{\textwidth}{!}{\begin{tabular}{|p{5cm}|p{5cm}|p{5cm}|}
+    \hline
+    Teammitglied & Summe der Bearbeiteten Story Points & Sonstige Beschreibung der Tätigkeit\\ \hline
+    Marcel Alex & 24 & N. A.\\
+    Jonas Eppard & 47 & Recomendation und ORM service Lead\\
```
Comment (eintim) [inline content/06kapitel.tex:126]: Jonas has 61 Points

File: content/06kapitel.tex [hunk 2]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+    Sophie Gösch & 15 & N. A.\\
+    Amos Groß & 54 & Projekt Organisation, Ersteller der User Stories und Product Owner\\
+    Tim Horlacher & 61 & Architekt und Core Service Lead, hat ein Großteil der Backend/Infra Stories Reviewed\\
+    Lukas Huida & 88 & Archtiekt, Organisation der \enquote{bare metal}-infra und server\\
+    Theo Krinitz & 19 & N. A.\\
+    Sabrina Ladner & 16 & Scrum Master\\
```
Comment (eintim) [inline content/06kapitel.tex:132]: Sabrina has 25 Points

File: content/06kapitel.tex [hunk 3]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+    \resizebox{\textwidth}{!}{\begin{tabular}{|p{5cm}|p{5cm}|p{5cm}|}
+    \hline
+    Teammitglied & Summe der Bearbeiteten Story Points & Sonstige Beschreibung der Tätigkeit\\ \hline
+    Marcel Alex & 24 & N. A.\\
+    Jonas Eppard & 47 & Recomendation und ORM service Lead\\
+    Sophie Gösch & 15 & N. A.\\
+    Amos Groß & 54 & Projekt Organisation, Ersteller der User Stories und Product Owner\\
+    Tim Horlacher & 61 & Architekt und Core Service Lead, hat ein Großteil der Backend/Infra Stories Reviewed\\
+    Lukas Huida & 88 & Archtiekt, Organisation der \enquote{bare metal}-infra und server\\
+    Theo Krinitz & 19 & N. A.\\
```
Comment (eintim) [inline content/06kapitel.tex:131]: Theo has 21 Points

File: content/06kapitel.tex [hunk 4]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+
+\begin{table}[h]
+    \resizebox{\textwidth}{!}{\begin{tabular}{|p{5cm}|p{5cm}|p{5cm}|}
+    \hline
+    Teammitglied & Summe der Bearbeiteten Story Points & Sonstige Beschreibung der Tätigkeit\\ \hline
+    Marcel Alex & 24 & N. A.\\
+    Jonas Eppard & 47 & Recomendation und ORM service Lead\\
+    Sophie Gösch & 15 & N. A.\\
+    Amos Groß & 54 & Projekt Organisation, Ersteller der User Stories und Product Owner\\
+    Tim Horlacher & 61 & Architekt und Core Service Lead, hat ein Großteil der Backend/Infra Stories Reviewed\\
```
Comment (eintim) [inline content/06kapitel.tex:129]: Tim has 64 Points

File: content/06kapitel.tex [hunk 5]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+Hierbei ist hervorzuheben, dass bei vielen Teammitgliedern Reviews und Fachliche Ausarbeitungen den Großteil deren Mitarbeit dargestellt hat, aber nicht mit in die Story Point Summe miteingeflossen ist. 
+Diese Sonstigen Beiträge sind jedoch in der entsprechenden Spalte in der Tabelle nachzulesen
+
+\begin{table}[h]
+    \resizebox{\textwidth}{!}{\begin{tabular}{|p{5cm}|p{5cm}|p{5cm}|}
+    \hline
+    Teammitglied & Summe der Bearbeiteten Story Points & Sonstige Beschreibung der Tätigkeit\\ \hline
+    Marcel Alex & 24 & N. A.\\
+    Jonas Eppard & 47 & Recomendation und ORM service Lead\\
+    Sophie Gösch & 15 & N. A.\\
```
Comment (eintim) [inline content/06kapitel.tex:127]: Sophie has 17 Points

File: content/06kapitel.tex [hunk 6]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+    Amos Groß & 54 & Projekt Organisation, Ersteller der User Stories und Product Owner\\
```
Comment (eintim) [inline content/06kapitel.tex:128]: Amos has 57 Points

File: content/06kapitel.tex [hunk 7]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
\ No newline at end of file
+Die Beiträge der einzelnen Personen sind in Tabelle \ref{tab:person_beitrag} nachzulesen.
+Hierbei ist hervorzuheben, dass bei vielen Teammitgliedern Reviews und Fachliche Ausarbeitungen den Großteil deren Mitarbeit dargestellt hat, aber nicht mit in die Story Point Summe miteingeflossen ist. 
+Diese Sonstigen Beiträge sind jedoch in der entsprechenden Spalte in der Tabelle nachzulesen
+
+\begin{table}[h]
+    \resizebox{\textwidth}{!}{\begin{tabular}{|p{5cm}|p{5cm}|p{5cm}|}
+    \hline
+    Teammitglied & Summe der Bearbeiteten Story Points & Sonstige Beschreibung der Tätigkeit\\ \hline
+    Marcel Alex & 24 & N. A.\\
```
Comment (eintim) [inline content/06kapitel.tex:125]: Marcel has 21 Points

File: content/06kapitel.tex [hunk 8]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+    Jonas Eppard & 47 & Recomendation und ORM service Lead\\
+    Sophie Gösch & 15 & N. A.\\
+    Amos Groß & 54 & Projekt Organisation, Ersteller der User Stories und Product Owner\\
+    Tim Horlacher & 61 & Architekt und Core Service Lead, hat ein Großteil der Backend/Infra Stories Reviewed\\
+    Lukas Huida & 88 & Archtiekt, Organisation der \enquote{bare metal}-infra und server\\
+    Theo Krinitz & 19 & N. A.\\
+    Sabrina Ladner & 16 & Scrum Master\\
+    Simon Morgenstern & 20 & Front-end Service Lead, hat ein Großteil der Frontend Stories Reviewed\\
```
Comment (eintim) [inline content/06kapitel.tex:133]: Simon has 25 Points

**Review by eintim — CHANGES_REQUESTED** [review]

File: content/06kapitel.tex [hunk 9]
```diff
@@ -100,5 +98,42 @@ \subsection{Ausgliederung der Estimations aus den Planungsbesprechungen}
...
+\begin{table}[h]
+    \resizebox{\textwidth}{!}{\begin{tabular}{|p{5cm}|p{5cm}|p{5cm}|}
+    \hline
+    Teammitglied & Summe der Bearbeiteten Story Points & Sonstige Beschreibung der Tätigkeit\\ \hline
+    Marcel Alex & 24 & N. A.\\
+    Jonas Eppard & 47 & Recomendation und ORM service Lead\\
+    Sophie Gösch & 15 & N. A.\\
+    Amos Groß & 54 & Projekt Organisation, Ersteller der User Stories und Product Owner\\
+    Tim Horlacher & 61 & Architekt und Core Service Lead, hat ein Großteil der Backend/Infra Stories Reviewed\\
+    Lukas Huida & 88 & Archtiekt, Organisation der \enquote{bare metal}-infra und server\\
```
Comment (eintim) [inline content/06kapitel.tex:130]: Lukas has 91

**Review by eintim — CHANGES_REQUESTED** [review]

File: content/06kapitel.tex (hunk 7 above)
Comment (eintim) [reply content/06kapitel.tex:125]: One story was assigned to amos nvm

File: content/06kapitel.tex (hunk 6 above)
Comment (eintim) [reply content/06kapitel.tex:128]: marcel did one story

**Review by Y05h129 — APPROVED** [review]

**Pull request merged**